sound/soc/sof/sof-priv.h

```diff
@@ -293,6 +293,7 @@ struct snd_sof_pcm_stream {
 	struct sof_ipc_stream_posn posn;
 	struct snd_pcm_substream *substream;
 	struct work_struct period_elapsed_work;
+	u32 avail_at_lp; /* pcm can run at low power like d0i3 */
```

Comment:
will be remove, and reuse the dai_link->ignore_suspend.

Reply:
OK, then let me give up that and keep using vendor token for it.

Reply:
> @keyonjie please squash with previous commit. otherwise you'll break git bisect with compilation errors with the previous commit.

indeed, thanks.

Reply:
@ranj063 actually this commit is before the token adding and token parsing ones in my personal branch:
https://github.com/keyonjie/linux/commits/pm_d0ix

don't know why the sequence is changed in the PR submitting, :-D

```
ASoC: SOF: topology: parse and store avail_at_lp flag …

@keyonjie
keyonjie committed 27 days ago
 
ASoC: SOF: token: add token for low power PCM. …

@keyonjie
keyonjie committed 27 days ago
 
ASoC: SOF: add flag to snd_sof_pcm_stream for low power stream …

@keyonjie
keyonjie committed 27 days ago
 
ASoC: SOF: Intel: hda-dsp: Add helper for setting dsp to d0i3 …

@keyonjie
keyonjie committed 28 days ago
```